library/src/main/java/com/shotcutter/library/config/MessageBrokerConfig.java

```diff
@@ -55,6 +56,11 @@ public RabbitTemplate rabbitTemplate(ConnectionFactory connectionFactory) {
         return template;
     }
 
+    @Bean
```

Comment:
do you need non-async template?

Reply:
Unfortunately yes, i'm still using it with authentication, so i'll rid of it when it would be completely refactored